src/index.js

```diff
@@ -56,16 +42,35 @@ async function main(params) {
   }
 }
 
-module.exports = {
-  main: wrap(openWhiskWrapper(main, {
+/**
+ * Runs the action by wrapping the `runExec` function with the pingdom-status utility.
+ * Additionally, if a EPSAGON_TOKEN is configured, the epsagon tracers are instrumented.
+ * @param params Action params
+ * @returns {Promise<*>} The response
+ */
+async function run(params) {
+  let action = runExec;
+  action = openWhiskWrapper(action, {
     token_param: 'EPSAGON_TOKEN',
     appName: 'Helix Services',
-    metadataOnly: false,
-    ignoredKeys: [/^[A-Z0-9_]+$/, 'token'],
-  }),
-  {
+    metadataOnly: false, // Optional, send more trace data
+    ignoredKeys: ['token', 'GOOGLE_PRIVATE_KEY', /[A-Z0-9_]+/],
```

Comment:
`GOOGLE_PRIVATE_KEY` is already covered by the regexp.

Reply:
Whoops great catch!